hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/contract/precompile/ERCPrecompileSuite.java

```diff
@@ -230,7 +232,51 @@ List<HapiSpec> erc721() {
                 someERC721OwnerOfScenariosPass(),
                 someERC721IsApprovedForAllScenariosPass(),
                 getErc721IsApprovedForAll(),
-                someERC721SetApprovedForAllScenariosPass());
+                someERC721SetApprovedForAllScenariosPass(),
+                erc20TransferFromDoesNotWorkIfFlagIsDisabled());
+    }
+
+    private HapiSpec erc20TransferFromDoesNotWorkIfFlagIsDisabled() {
+        return defaultHapiSpec("erc20TransferFromDoesNotWorkIfFlagIsDisabled")
+                .given(
+                        overriding("hedera.allowances.isEnabled", "false"),
+                        newKeyNamed(MULTI_KEY),
+                        cryptoCreate(OWNER).balance(100 * ONE_HUNDRED_HBARS),
+                        cryptoCreate(RECIPIENT),
+                        cryptoCreate(TOKEN_TREASURY),
+                        tokenCreate(FUNGIBLE_TOKEN)
+                                .tokenType(TokenType.FUNGIBLE_COMMON)
+                                .supplyType(TokenSupplyType.FINITE)
+                                .initialSupply(10L)
+                                .maxSupply(1000L)
+                                .treasury(TOKEN_TREASURY)
+                                .adminKey(MULTI_KEY)
+                                .supplyKey(MULTI_KEY),
+                        uploadInitCode(ERC_20_CONTRACT),
+                        contractCreate(ERC_20_CONTRACT),
+                        tokenAssociate(OWNER, FUNGIBLE_TOKEN),
+                        tokenAssociate(RECIPIENT, FUNGIBLE_TOKEN),
+                        tokenAssociate(ERC_20_CONTRACT, FUNGIBLE_TOKEN),
+                        cryptoTransfer(moving(10, FUNGIBLE_TOKEN).between(TOKEN_TREASURY, OWNER)))
+                .when(withOpContext((spec, opLog) -> allRunFor(
+                        spec,
+                        contractCall(
+                                        ERC_20_CONTRACT,
+                                        TRANSFER_FROM,
+                                        HapiParserUtil.asHeadlongAddress(
+                                                asAddress(spec.registry().getTokenID(FUNGIBLE_TOKEN))),
+                                        HapiParserUtil.asHeadlongAddress(
+                                                asAddress(spec.registry().getAccountID(OWNER))),
+                                        HapiParserUtil.asHeadlongAddress(
+                                                asAddress(spec.registry().getAccountID(RECIPIENT))),
+                                        BigInteger.TWO)
+                                .gas(500_000L)
+                                .via(TRANSFER_FROM_ACCOUNT_TXN)
+                                .hasKnownStatus(CONTRACT_REVERT_EXECUTED))))
+                .then(
+                        getTxnRecord(TRANSFER_FROM_ACCOUNT_TXN)
+                                .logged(), // has gasUsed little less than supplied 500K in contractCall result
```

Comment:
I'm not familiar with this test suite - but what does this comment mean?  is that something that should be checked, or already is checked, here or elsewhere?

Reply:
We are checking if the gasUsed is almost same as submitted gas

Reply:
i guess i don't see _where_ that test is made - like I said, I'm not familiar with the test suite